explorer/models/containerNode.ts

```diff
@@ -21,14 +22,8 @@ export class ContainerNode extends NodeBase {
     }
 
     public getTreeItem(): vscode.TreeItem {
-        let displayName: string = this.label;
-
-        if (vscode.workspace.getConfiguration('docker').get('truncateLongRegistryPaths', false)) {
-            if (/\//.test(displayName)) {
-                let parts: string[] = this.label.split(/\//);
-                displayName = trimWithElipsis(parts[0], vscode.workspace.getConfiguration('docker').get('truncateMaxLength', 10)) + '/' + parts[1];
-            }
-        }
+        let config = vscode.workspace.getConfiguration('docker');
+        let displayName: string = getImageOrContainerDisplayName(this.label, config.get('truncateLongRegistryPaths'), config.get('truncateMaxLength', 10));
```

Comment:
`config.get('truncateLongRegistryPaths', false)` was the previous usage. I see package.json properly handles the default in this case. 
Looking at package.json again, the 10 in `config.get('truncateMaxLength', 10)` should not be required in any of these calls.